Client/Frontend/Browser/SearchViewController.swift

```diff
@@ -309,9 +300,9 @@ class SearchViewController: SiteTableViewController,
             tableView.topAnchor.constraint(equalTo: view.topAnchor),
             tableView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
             tableView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
-            // Ecosia: Quick Search Shortcuts Experiment
+            // Ecosia: Hide `searchEngineScrollView` by changing constraints
             // tableView.bottomAnchor.constraint(equalTo: searchEngineScrollView.topAnchor)
-            tableView.bottomAnchor.constraint(equalTo: tableViewBottomEqualToAnchor)
+            tableView.bottomAnchor.constraint(equalTo: view.bottomAnchor)
```

Comment:
Instead of re-adding the commented code from before the experiment  (see [these changes](https://github.com/ecosia/ios-browser/pull/596/files#diff-ed4bcd35f5f4f84b493b928963096a772b0f48dbfbf7ff0d91416db6aaebabd4)), I thought it would be cleaner and more maintainable to just keep the same behaviour we've had for a while with the experiment turned off. This basically still keeps and sets up `searchEngineScrollView` but hides it via constraints.

Reply:
Thanks for adding the comment here to keep track of the conversation (resulting in agreement 👍 ) we had.